### ISSUE TITLE
Ability to generate everything based Domains.

### ISSUE THREAD

**Issue by Erth0** [laravel-shift/blueprint#96]
### It would be good if we could generate everything which is currently supported under Domain section what i mean is something like this.

    domains:
	  Post:
	    models:
	      Post:
	        title: string:400
	        content: longtext
	        published_at: nullable timestamp

	    controllers:
	      Post:
	        index:
	          query: all
	          render: post.index with:posts

	        store:
	          validate: title, content
	          save: post
	          send: ReviewNotification to:post.author with:post
	          dispatch: SyncMedia with:post
	          fire: NewPost with:post
	          flash: post.title
	          redirect: post.index

	  Comment:
	    models:
	      Comment:
	        content: longtext
	        published_at: nullable timestamp

	    controllers:
	      Comment:
	        store:
	          validate: title, content
	          save: post
	          send: ReviewNotification to:post.author with:post
	          dispatch: SyncMedia with:post
	          fire: NewPost with:post
	          flash: post.title
	          redirect: post.index
Basically everything will be generated as listed in this youtube video by Freek [Flare, OhDear](https://www.youtube.com/watch?v=A7JEfsUeey0&feature=emb_title)

**Comment by jasonmccreary** [comment]
This will likely not be something I build into Blueprint.

However, you could achieve the desired result by changing the `path` in your `config/blueprint.php` file.
